### PR TITLE
Configure users module packages after installation

### PR DESCRIPTION
**Pull request by jvalleroy**
Preseeding settings with debconf won't have any effect if the packages are already installed. Instead, provide an override database to dpkg-reconfigure.

Note that in the dpkg-reconfigure step, even if it's successful it will return an error code to indicate that it could not write to the override database file (because of the readonly flag). That's why I did not set check=True for this command.

I've tested an image that included this change, along with adding the users module's required packages as dependencies for freedombox-setup. I confirmed that users module and user logins were working as expected.

### CODE REVIEW

File: actions/users [hunk 1]
```diff
@@ -185,6 +161,28 @@ def configure_ldapscripts():
...
+Value: {value}
+Owners: {package}
+'''
+    override_data = ''
+    for key, value in config.items():
+        override_data += override_template.format(
+            package=package, key=key, value=value)
+
+    with open('/tmp/override.dat', 'w') as override_file:
+        override_file.write(override_data)
```
Comment (SunilMohanAdapa) [inline actions/users]: mktemp.mkstemp() should be used here.

**Comment by jvalleroy** [comment]
I added a commit to use tempfile.NamedTemporaryFile(). This seemed a bit simpler than tempfile.mkstemp().

**Comment by SunilMohanAdapa** [comment]
I have merged this patch.  Did a lot of testing.

I tried to unsuccessfully use the Pipe DbDriver instead of File DbDriver.  Pipe{readonly:true} worked mostly except for nslcd package for which the Debconf initialization was happening twice.  Since data from stdin can't be read twice, the second initialization was not getting any data.

Waiting on fresh build test report to close this issue.

**Comment by SunilMohanAdapa** [comment]
Did a full build test on amd64.  Things seems okay with the ldap setup part (although I experienced a problem where a lot of services including plinth, apache2, and freedombox first boot are not starting up).

**Pull request closed without merging**